src/main/java/com/quizapp/enterprise/events/GameOverEventListener.java

```diff
@@ -16,6 +16,8 @@
     @EventListener
     @MessageMapping("/chat")
     public void onApplicationEvent(GameOverEvent event) {
-        messagingTemplate.convertAndSend("/game1/gameOver/" + event.gameCode, event.leaderboard);
+        StringBuilder gameOverCodeString = new StringBuilder("/game1/gameOver/")
+                .append(event.gameCode);
+        messagingTemplate.convertAndSend(gameOverCodeString.toString(), event.leaderboard);
```

Comment:
refactored the string concatenation to use String builder with append.
This change was recommended to by "5. Use StringBuilder or StringBuffer for String Concatenation " section of the java coding best practices summary referenced below.

source: https://xperti.io/blogs/java-coding-best-practices/